### DISCUSSION TITLE
Setup external PostgreSQL from dump and connect docker nominatim to it

### DISCUSSION THREAD

**Discussion by dmestudent** [mediagis/nominatim-docker#556] [General]
Hello,

I`d like to ask if it is possible to dump our on premise hosted nominatim database with pg_dump, insert the dump into a PostgreSQL database hosted in Azure and then configuring the nominatim docker container so that its using the external Azure PostgreSQL Database without trying to initialize it from scratch. 

I read through a bunch of github issues where people discussed various issues with Azure which should not be the part of this issue. I just added the Azure information for context. We can also break this issue down to the question is it possible to have the Nominatim Database on one server, dump the database, insert the database dump on a new server, change the external PostgreSQL connection values for the docker containe, have it connect to the database on the new server and work as before without any problem. 

Thank you for your time and this great project.
Best regards

**Comment by leonardehrenfried** [comment]
In theory it should work but be aware that using an external postgres db is not supported, so if it works, it just happens to and isn't necessarily designed that way. That said, there seems to be a number of people who do that quite successfully.
